packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -155,6 +155,9 @@ export type BasicMeetingInformation = {
   };
   meetingInfo: any;
   sessionCorrelationId: string;
+  roles: string[];
```

Comment:
question. Why have we also added roles and callStateForMetrics?

Reply:
roles being read in getCurUserType utility function and callStateForMetrics is being read in next line where previous error occured.